.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Release Checklist
```

Comment:
We should rename this to feature freeze release checklist or include that wording in the issue.

Reply:
I think here it is fine, but I also added this suggestion as a main header in the doc.